roles/conjur_host_identity/tests/Dockerfile

```diff
@@ -40,3 +40,7 @@ RUN apt-get update && \
 RUN apt-get update && apt-get install -y gcc build-essential
 RUN apt-add-repository -y ppa:brightbox/ruby-ng && apt-get update && apt-get install -y ruby2.4 ruby2.4-dev
 RUN gem install conjur-cli
+
+# install Ansible Conjur Collection
+RUN ansible-galaxy collection install cyberark.conjur && \
+    ansible-galaxy install cyberark.conjur-host-identity
```

Comment:
I'm not so sure about this, some considerations...
1. This repo should really be testing its own source code, instead of what is on Ansible Galaxy
2. The standalone role is separate to the collection, and I think any test for it should reside in the standalone role's repo. All that to maintain a separation of concerns.

Reply:
For (1) I think if we want a smoke test to ensure that we can always install the role from galaxy then we should define that separate to the tests that currently exist.